src/jdk.javadoc/share/classes/jdk/javadoc/internal/html/HtmlTree.java

```diff
@@ -1155,6 +1157,15 @@ public static HtmlTree WBR() {
         return WBR_INSTANCE;
     }
 
+    /**
+     * {@return an HTML {@code IMG} element}
+     */
```

Comment:
It would be nice to also document/mention `src` and `alt` parameters/atttributes.